package.json

```diff
@@ -36,10 +34,8 @@
     "lint:css:scss": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" --custom-syntax postcss-scss",
     "lint:css:scss:color": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" \"!source/sass/**/_colors.scss\" \"!source/sass/**/formassembly-override.scss\" --custom-syntax postcss-scss --config .stylelintrc-colors.js",
     "lint:css:network": "stylelint \"network-api/networkapi/{,!(frontend)/**/}*.css\" --custom-syntax postcss-scss",
-    "optimize:css": "run-s optimize:css:*",
-    "optimize:css:run": "postcss network-api/networkapi/frontend/_css/*.css --dir network-api/networkapi/frontend/_css/temp",
-    "optimize:css:copy": "shx mv network-api/networkapi/frontend/_css/temp/* network-api/networkapi/frontend/_css",
-    "optimize:css:clean": "shx rm -rf network-api/networkapi/frontend/_css/temp",
+    "postcss:build": "postcss network-api/networkapi/temp/frontend/_css/*.css --dir network-api/networkapi/frontend/_css --verbose",
```

Comment:
Add a new script `postcss:build`. This is to use `postcss` to process compiled CSS files with the plugins we specified in `postcss.config.js`